plugins/backend/catalog-backend-module-okta/src/providers/OktaGroupEntityProvider.ts

```diff
@@ -55,17 +51,9 @@ export class OktaGroupEntityProvider extends OktaEntityProvider {
     const orgUrl = config.getString('orgUrl');
     const token = config.getString('token');
 
-    const userFilters = userFiltersFromConfigArray(
-      config.getOptionalConfigArray('userFilters'),
-    );
-    const groupFilters = userFiltersFromConfigArray(
-      config.getOptionalConfigArray('groupFilters'),
-    );
+    const groupFilter = config.getOptionalString('groupFilter');
```

Comment:
the config here will be the root config now, no ? So I think these needs to be something like: `catalog.providers.okta.groupFilter`

Reply:
I only changed the root config for the new one I have created. I didn't change this. It would be a breaking change to change this for the existing ones.

I would prefer not to change this because its not within the scope of what I am doing here. I can revisit it after as part of a separate effort if this is what we want?